security-admin/src/main/webapp/scripts/mgrs/RangerConfigMgr.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+// Manages the ranger configuration
+define(function(require){
+	'use strict';
+
+	var RangerManagementConfig = require('models/RangerManagementConfig');
+
+
+	// Private properties
+	var rangerManagementConfig = null;
+
+	// Public methods
+	var RangerConfigMgr = {};
+
+	/**
+	 * Gets the ranger configuration  manager
+	 *
+	 * @returns RangerConfigMgr
+	 */
+	RangerConfigMgr.getRangerManagementConfig = function() {
+		if ( rangerManagementConfig){
+			return rangerManagementConfig;
```

Comment:
Please use 4-space indentation for JavaScript here and below.
@raviprak-altiscale @zoran-altiscale do we use the same Coding Standards (https://altiscale.xwiki.com/xwiki/bin/view/product-development/coding-standards#Attachments) for community code as well?

Reply:
this is open source. I checked ranger style checks - it passes. However, I would still prefer if Cetin is consistent, but the main style guide for this is what Ranger team wants, not altiscale.